state-chain/pallets/cf-elections/src/electoral_systems/mocks/access.rs

```diff
@@ -110,8 +110,12 @@ impl<ES: ElectoralSystem> ElectionWriteAccess for MockWriteAccess<ES> {
 		new_extra: <Self::ElectoralSystem as ElectoralSystem>::ElectionIdentifierExtra,
 		properties: <Self::ElectoralSystem as ElectoralSystem>::ElectionProperties,
 	) -> Result<(), CorruptStorageError> {
+		// Remove the old properties and replace it with the new one.
+		MockStorageAccess::set_election_properties::<ES>(self.identifier(), None);
```

Comment:
we don't need to explicitly remove, it'll get overwritten by setting to the new properties

Reply:
Same as before - the "extra" will just add a new entry instead of overwriting the old one, since Extra has changed.